### PR TITLE
New style

### PR DESCRIPTION
**Pull request by stephanlindauer**
@zutrinken moinsen! wollte mal fragen was wir mit den aenderungen auf dem branch am besten machen. ist das schon mergeable?

### CODE REVIEW

**Comment by zutrinken** [comment]
@stephanlindauer yo, das kann übernommen werden wenns gefällt.

**Comment by headione** [comment]
great design IMHO, thanks for your work @zutrinken ! @stephanlindauer do you also like the new style? the color of the buttons reminds me a bit to spotify ;) but it gives a nice contrast to the background.

File: src/css/_base.scss [hunk 1]
```diff
@@ -0,0 +1,234 @@
+html {
+	font-size: 62.5%;
+}
+
+body {
+	font-family: 'Open Sans', sans-serif;
+	font-weight: 300;
+    font-size: 1.5em;
```
Comment (headione) [inline src/css/_base.scss]: minor issues with indentation here... we can use linter in future. e.g. scss-lint

File: src/css/style.scss [hunk 2]
```diff
@@ -1,5 +1,46 @@
-$breakpoint-phone:	30em;
-$breakpoint-tablet:	50em;
-$breakpoint-laptop:	70em;
+@font-face {
```
Comment (headione) [inline src/css/style.scss]: we could extract the font descriptions to something like _fonts.scss

File: src/css/style.scss [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+    font-family: 'Open Sans';
+    font-style: italic;
+    font-weight: 400;
+    src: local('Open Sans Italic'), local('OpenSans-Italic'), url('https://fonts.gstatic.com/s/opensans/v13/xjAJXh38I15wypJXxuGMBrrIa-7acMAeDBVuclsi6Gc.woff') format('woff');
+}
+
+
+
+$color-green:			#3DCC85;
```
Comment (headione) [inline src/css/style.scss]: We could extract these vars to something like _vars.scss IMHO

**Comment by stephanlindauer** [comment]
woah, sieht mega fett aus! ich schaus mir spaeter mal genauer an.

**Comment by zutrinken** [comment]
added your suggestions

**Comment by headione** [comment]
Yellow? really? :/ this doesn't look fresh like with the green (or the blue color in the iOS app ;)

**Comment by headione** [comment]
thanks for the changes... i don't like the yellow, but thats just my taste. i'll implement a linter soon on a different branch

**Comment by zutrinken** [comment]
@headione haha, k. was just looking for a strong color on the dark background. Actually I'd prefer to make progress on the app interface instead to get a unified look ...

**Comment by stephanlindauer** [comment]
will deploy on the weekend. currently infrastructure setup and new api is still a mess. :|

**Pull request merged**